packages/babel-plugin-transform-function-name/src/index.ts

```diff
@@ -20,7 +25,7 @@ export default declare(api => {
       ObjectProperty(path) {
         const value = path.get("value");
         if (value.isFunction()) {
-          const newNode = nameFunction(value);
+          const newNode = nameFunction(value, null, supportUnicodeId);
```

Comment:
```suggestion
          const newNode = nameFunction(value, undefined, supportUnicodeId);
```
or
```suggestion
          const newNode = nameFunction(value, false, supportUnicodeId);
```

with `undefined` it will get the default value (`false`); with `null` it won't.

Reply:
I will the change the following in the next commit.